scss/colors.module.scss

```diff
@@ -6,3 +6,19 @@ $secondary: #fff;
 $warning: #ffae28;
 $green: #00cf92;
 $textDanger: #721c24;
+
+//  BACKGROUND
+
+// Used to identify main call to action buttons.
+// Also used as selected tab color and table header background color
+$purple: hsl(248, 66%, 55%);
+// Used as secondary color and primarily for save buttons
+$green: hsl(170, 66%, 38%);
+// Used for page backgrounds
+$light-gray: hsl(0, 0%, 96%);
+// Used for menu item background color on hover
+$light-purple: hsl(232, 100%, 97%);
+// Used for header background color
+$white: hsl(0, 0%, 100%);
+// Use to be defined
+$dark-gray: hsl(248, 10%, 42%);
```

Comment:
There is already a darkgray color variable, wouldn't it be confusing to have two?

Reply:
Oh right

Reply:
Going to approve it, so you can merge it whenever it is updated